src/components/Dialog/__stories__/Dialog.stories.scss

```diff
@@ -0,0 +1,26 @@
+@import "../../../styles/themes.scss";
```

Comment:
did not implement it with css modules because we have an issue with typescript+css modules in stories, tried to understand what is not working but did not want to waste time on this because out of this task scope